plutus-playground/plutus-playground-client/src/Types.purs

```diff
@@ -159,12 +159,15 @@ data Query a
   | RemoveWallet Int a
   | SetBalance Wallet Int a
   -- Actions.
-  | AddAction Action a
-  | AddWaitAction Int a
-  | RemoveAction Int a
+  | ModifyActions ActionEvent a
```

Comment:
don't you think `ModifyAction` rather than `ModifyActions`?

Reply:
While we only have `ActionEvent`s that modify/add/remove a single action, this type doesn't know that, or enforce it. We could easily have an `ActionEvent` called `Filter`, for example. So the plural is correct IMO.